backend/server.py

```diff
@@ -143,21 +141,24 @@ async def router(self, websocket: WEBSOCKET_TEMP_TYPE, path: str) -> None:  # py
             init_message = json.loads(await websocket.recv())  # can throw error, will just hit the unregister
             print(init_message)
             if init_message.get('msg') == 'authTokenVerification':
-                user = self.verify_websocket_auth(init_message['data'])  # can throw error
-                await self.register(websocket, user)
+                try:
+                    user = self.verify_websocket_auth(init_message['data'])  # can throw error
+                    await self.register(websocket, user)
+                except Exception as e:
+                    await websocket.close()
             else:
                 await websocket.close()
-        try:
-            async for message in websocket:
-                data = json.loads(message)
-                print(data)
-                # if data['type'] == MessageType.USER_JOINED.value:
-                #     await handle_new_user_joined(websocket, data)
-                # elif data['type'] == 'getUsernames':
-                #     await handle_get_usernames(websocket)
-        finally:
-            await self.unregister(websocket)
-
+        else:
+            try:
+                async for message in websocket:
+                    data = json.loads(message)
+                    print(data)
+                    # if data['type'] == MessageType.USER_JOINED.value:
+                    #     await handle_new_user_joined(websocket, data)
+                    # elif data['type'] == 'getUsernames':
```

Comment:
Why `getUsernames` not in `MessageType`?

Reply:
all this was like pre-alpha POC code and will be deleted/replaced as we implement the stories that need to send these IPC messages